### PR TITLE
Use custom meta title ONLY if there is one.

### PR DESCRIPTION
**Pull request by hotdoy**

### CODE REVIEW

File: workspace/utilities/lib/master-title.xsl [hunk 1]
```diff
@@ -38,12 +38,10 @@
...
 		
-		<xsl:if test="string-length($cur-page-title) != 0 and count(/data/params/page-types/item[@handle = 'index']) = 0">
-			<xsl:value-of select="$cur-page-title"/>
-			<xsl:text> - </xsl:text>
-		</xsl:if>
-		
 		<xsl:choose>
+			<xsl:when test="string-length($cur-page-title) != 0 and count(/data/params/page-types/item[@handle = 'index']) = 0">
+				<xsl:value-of select="$cur-page-title"/>
+			</xsl:when>
```
Comment (nitriques) [inline workspace/utilities/lib/master-title.xsl]: Should we allow for the home page to have custom metas ? Not necessarily the default one ? @PascalPiche @fhamon @hotdoy @beaubbe

Comment (nitriques) [inline workspace/utilities/lib/master-title.xsl]: @PascalPiche @fhamon @hotdoy @beaubbe ping!

Comment (hotdoy) [inline workspace/utilities/lib/master-title.xsl]: Maybe we could.

Comment (nitriques) [inline workspace/utilities/lib/master-title.xsl]: I don't. If we stop to merge, we need to stop to append also. @PascalPiche @beaubbe @fhamon

**Comment by nitriques** [comment]
I'd like @PascalPiche to review this as well.

**Pull request closed without merging**